common/source/docs/common-polygon_fence.rst

```diff
@@ -1,64 +1,133 @@
+[/site]
+[site wiki="plane"]
+Like the :ref:`cylindrical fence <common-ac2_simple_geofence>`, Plane will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` if the boundary or altitude limits are breached.
+[/site]
 
-See :ref:`cylindrical fence <common-ac2_simple_geofence>` for fence parameters, which also apply to these polygonal fences.
+These fences are created and treated in a similar way to mission command lists and rally point lists and loaded into the autopilot. Each item in the list specifies a point in the polygonal boundary or a circular item, whether its inclusion or exclusion type, and radius if circular.
+[site wiki="plane"]
+For Plane a fence breach return point can also be specified in the list.
+[/site]
 
-These fences are created and treated in a similar way to mission command lists and rally point lists and loaded into the autopilot.
+Multiple fences can be specified of differing types and shapes in the list.
 
 
 ..  youtube:: U3Z8bO3KbyM
     :width: 100%
 
+
+
 Enabling Fences in Mission Planner
 ==================================
 
-..  youtube:: SEm4nVfbg00
-    :width: 100%
+To enable Fences, go to the Mission Planner full parameter list (CONFIG->Full Parameter Tree), search for items with ``FENCE_``:
 
--  Connect your Autopilot to the Mission Planner
--  Go to the **CONFIG\| GeoFence** screen
--  Click "Enable" and set Type to "All" or another option that includes "Polygon". Circle refers to the HOME centered, :ref:`cylindrical fence <common-ac2_simple_geofence>`. Altitude only, limits maximum altitude (everywhere).
+.. image:: ../../../images/fence_enable.png
+    :target: ../_images/fence_enable.png
 
+Plane Parameter List (Copter/Rover have fewer)
 
-   .. image:: ../../../images/polygon_fence_enable.png
-       :target: ../_images/polygon_fence_enable.png
-       :width: 500px
+-  Set :ref:`FENCE_ACTION<FENCE_ACTION>` = to whatever you wish for a breach action. These will vary depending on vehicle type.
+-  Set :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` = to the altitude limit you want (in meters). This is unavailable in Rover. A value of 0 disables this limit.
+-  Set :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1 to enable FENCEs.
+-  Set :ref:`FENCE_MARGIN<FENCE_MARGIN>` = to the distance from the fence horizontal boundary the vehicle must maintain in order to prevent a breach.
+-  Set :ref:`FENCE_RADIUS<FENCE_RADIUS>` to the maximum distance from HOME you want (in meters). This should normally be at least 50m. This value must be larger than :ref:`FENCE_MARGIN<FENCE_MARGIN>` and greater than 30m.
+-  :ref:`FENCE_TOTAL<FENCE_TOTAL>` should not be changed manually and indicates the number of inclusion/exclusion fence points stored. (See :ref:`common-polygon_fence`)
+-  Set :ref:`FENCE_TYPE<FENCE_TYPE>` = default of "ALL" is fine and will allow you to set up an additional :ref:`common-ac2_simple_geofence` , unless you want only a the inclusion/exclusion fences to be active if a home based cylindrical fence has also been enabled.
```

Comment:
Again, default is not set to ALL - ever.
Rover defaults to Circle + Polygon.
Plane defaults to Polygon.
All others default to Alt Max + Tin Can + Polygon.

We still have ALT Min available, that's not used by default

Reply:
corrected